cpp/oneapi/dal/io/detail/load_graph.cpp

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+//#include <algorithm>
+
+#include "oneapi/dal/io/detail/load_graph.hpp"
+
+#include "oneapi/dal/detail/common.hpp"
+#include "oneapi/dal/exceptions.hpp"
+#include "oneapi/dal/graph/detail/container.hpp"
+#include "oneapi/dal/graph/detail/undirected_adjacency_vector_graph_impl.hpp"
+#include "oneapi/dal/graph/common.hpp"
+#include "oneapi/dal/common.hpp"
+#include "oneapi/dal/graph/undirected_adjacency_vector_graph.hpp"
+#include "oneapi/dal/io/detail/load_graph_service.hpp"
+#include "oneapi/dal/detail/threading.hpp"
+#include "oneapi/dal/io/graph_csv_data_source.hpp"
+#include "oneapi/dal/io/load_graph_descriptor.hpp"
+#include "services/daal_atomic_int.h"
```

Comment:
need to think if this classic daal headers can be included here

Reply:
Anyway, all DAAL headers are included as <...> with full path. For example, see `cpp/oneapi/dal/backend/interop/common.hpp`